.github/ISSUE_TEMPLATE/bug_report_site_issue.md

```diff
@@ -9,6 +9,9 @@ assignees: ''
 
 **Affected website:** 
 
+<-- Tick the following box, if the issue is severe and breaks the whole site or input element. -->
+* [ ] This issue makes the site/input field completly unusable.
```

Comment:
Minor spelling issue: completly → completely

Reply:
Thanks. Next time, you can BTW, also use GitHub's propose feature to directly propose the change and I can apply it with one click. :slightly_smiling_face: